docker-compose-prod.yml

```diff
@@ -35,33 +38,32 @@ services:
       - ./fileserver/uploads:/root/uploads
 
   nginx:
-    image: linuxserver/letsencrypt
+    image: linuxserver/swag
     restart: "always"
     ports:
       - "80:80"
       - "443:443"
     environment:
-      - PUID=1013
-      - GUID=1015
+      - PUID=1002
+      - GUID=1003
       - EMAIL=it@nettverksdagene.no
       - URL=nettverksdagene.no
-      - SUBDOMAINS=www, blogg
-      - EXTRA_DOMAINS=nvdagen.no,www.nvdagen.no
+      - SUBDOMAINS=www,blogg
       - TZ=Europe/Oslo
       - VALIDATION=http
       #- STAGING=true # Uncomment this when testing
     depends_on:
       - api
       - fileserver
-      - deploy
+      #- deploy
     volumes:
       - ./nginx/config:/config # Todo: Can this volume include the below?
       - frontend-files:/frontend
 
-  deploy:
-    build: ./deploy
-    restart: "always"
-    environment:
-      - GITHUB_WEBHOOKS_SECRET=${GITHUB_WEBHOOKS_SECRET}
+  #deploy:
+  #  build: ./deploy
+  #  restart: "always"
+  #  environment:
+  #    - GITHUB_WEBHOOKS_SECRET=${GITHUB_WEBHOOKS_SECRET}
```

Comment:
Shouldn't we delete this instead of just commenting it out? git stores history anyways, and this clutters the files.